### PR TITLE
Remove all 'any' or mistyped values

### PR DESCRIPTION
**Pull request by vtkalek**

### CODE REVIEW

File: src/radarChart.ts [hunk 1]
```diff
@@ -313,7 +243,7 @@ module powerbi.extensibility.visual {
 
             for (let i: number = 0, iLen: number = categoryValues.length; i < iLen; i++) {
                 labelsData.labelPoints.push({
-                    text: categoryValues[i],
+                    text: categoryValues[i].toString(),
```
Comment (ignatvilesov) [inline src/radarChart.ts]: Is it possible that `categoryValues[i]` might be `null`? I suppose that it'd be better to use type casting: `categoryValues[i] as string`

File: src/radarChart.ts [hunk 2]
```diff
@@ -367,24 +286,24 @@ module powerbi.extensibility.visual {
                 series: RadarChartSeries[] = [],
                 grouped: DataViewValueColumnGroup[],
                 colorHelper: ColorHelper;
-
+            const settings: RadarChartSettings = this.parseSettings(dataView, colorPalette);
```
Comment (ignatvilesov) [inline src/radarChart.ts]: Why do we need to parse settings twice? Please take a look at line#278

File: src/radarChart.ts [hunk 3]
```diff
@@ -367,24 +286,24 @@ module powerbi.extensibility.visual {
...
             grouped = catDv && catDv.values
                 ? catDv.values.grouped()
                 : null;
-
-            colorHelper = new ColorHelper(colorPalette, RadarChart.Properties.dataPoint.fill);
+            const fillProp: DataViewObjectPropertyIdentifier = {
```
Comment (ignatvilesov) [inline src/radarChart.ts]: That's ok. However, it'd be better to more it to a static variable.

File: src/radarChart.ts [hunk 4]
```diff
@@ -412,6 +331,7 @@ module powerbi.extensibility.visual {
                     }
 
                     if (source.objects) {
+                        debugger;
```
Comment (ignatvilesov) [inline src/radarChart.ts]: Please remove this line

File: src/radarChart.ts (hunk 2 above)
Comment (vtkalek) [inline src/radarChart.ts]: It's not a twice parsing - at row 278 parse used in return defenition

File: src/radarChart.ts (hunk 3 above)
Comment (vtkalek) [inline src/radarChart.ts]: done

**Pull request merged**